### DISCUSSION TITLE
afterMethod displays in the allure report when a test is broken

### DISCUSSION THREAD

**Discussion by kabothu** [allure-framework/allure2#2206] [General Discussion]
I'm using Playwright Java. I'm also using Testng Listeners. Whenever my test is broken, @AfterMethod displays in the allure report which has tear down methods.When ever this gets added, the test count gets increased by 1. Can you please let me know how to ignore this showing in the allure report?

**Comment by baev** [comment]
You can opt-out from displaying TestNG configuration failures in the report by setting `allure.testng.hide.configuration.failures` property to false in `allure.properties` 

```properties
allure.testng.hide.configuration.failures=false
```
